web/src/compose_call_ui.ts

```diff
@@ -87,7 +106,7 @@ export function generate_and_insert_audio_or_video_call_link($target_element, is
                             );
                         }
                     },
-                }),
+                })!,
```

Comment:
What makes this safe?

Reply:
`channel.post` may return undefined. What should we do if we don't want to store undefined in the map? I am a bit confused here. Should we `assert` this?

Reply:
Look at the code of `channel.post` and ask yourself *why* it might return `undefined`.

- Is it an error condition that should never happen? Use `assert`.
- Is it a normal absence of a value? Use an `if` statement.
- Are you an agent of chaos seeking to sow confusion by continuing execution with a value whose TypeScript type doesn’t match its runtime type? Use `!`.

(Hint: `if`.)

Reply:
Thank you so much for the detailed explanation.